libgdf/include/gdf/arrow.hpp

```diff
@@ -0,0 +1,6 @@
+#include "cffi/types.h"
+#include <arrow/api.h>
+
```

Comment:
Please add block comments as in this file https://github.com/rapidsai/cudf/blob/3c7bea328533a78907ab25871d10095e9a838c87/libgdf/src/example_documentation.cpp (not sure what happened to this file in the refactor)